gslib/addlhelp/dev.py

```diff
@@ -117,20 +117,10 @@
   9. Please consider contributing test code for your change, especially if the
      change impacts any of the core gsutil code (like the gsutil cp command).
 
-  10. When it's time to send us code, please use the Rietveld code review tool
-      rather than simply sending us a code patch. Do this as follows:
-
-      - Check out the gsutil code from your fork of the gsutil repository and
-        apply your changes.
-      - Download the "upload.py" script from
-        https://github.com/rietveld-codereview/rietveld
-      - Run upload.py from your git directory with the changes.
-      - Click the codereview.appspot.com link it generates, click "Edit Issue",
-        and add mfschwartz@google.com and thobrla@google.com as reviewers, and
-        Cc gs-team@google.com.
-      - Click Publish+Mail Comments.
-      - Once your changes are accepted, submit a pull request on GitHub and we
-        will merge your commits.
+  10. When it's time to send us code, please submit a PR to the [gsutil GitHub
```

Comment:
Nit: Link style is different in .py files than in .md files

Reply:
Updated link formatting :)